pymc3/tests/test_distributions_random.py

```diff
@@ -539,6 +539,23 @@ def test_probability_vector_shape(self):
         assert pm.Categorical.dist(p=p).random(size=4).shape == (4, 3, 7)
 
 
+class TestDirichlet(SeededTest):
+    def test_dirichlet_random_shape(self):
+        assert pm.Dirichlet.dist(a=np.ones(2)).random(size=(1)).shape == (1, 2)
```

Comment:
You can make use of `pytest.mark.parametrize` for different combinations of `a` and `random` shapes.

Reply:
Yeah, I think that would be better.